app/controllers/Users.php

```diff
@@ -31,7 +31,8 @@ public function signup() {
             if($_SERVER['REQUEST_METHOD'] == 'POST') {
                 // sanitize post data
                 // filter_input_array() returns false if POST is set to scalar value
-                $_POST = filter_input_array(INPUT_POST, FILTER_SANITIZE_STRING);
+                // $_POST = filter_input_array(INPUT_POST, FILTER_SANITIZE_STRING);
+                $_POST = filter_input_array(INPUT_POST);
```

Comment:
Can you explain the reason for removing FILTER_SANITIZE_STRING in $_POST = filter_input_array(INPUT_POST, FILTER_SANITIZE_STRING);